### DISCUSSION TITLE
change logo and pagesidebar buttom

### DISCUSSION THREAD

**Discussion by supraxd2021** [mediacms-io/mediacms#691] [Q&A]
hi,

please help me ,how to change a logo and edit page sidebar buttom.
where can i find path to change

Thanks for help

**Comment by Micrufun** [comment]
To change logo, you may simply replace these files:

```
static/images/logo_dark.png
static/images/logo_dark.svg
static/images/logo_light.png
static/images/logo_light.svg
```

**Reply by supraxd2021** [reply]
i already change the picture ,but in web still mediacms logo
please advice more.thanks

**Reply by tim535** [reply]
Try to open the page in an incognito tab of your browser. This helped for me...

**Reply by indangasy** [reply]
How do i edit the logo size? Managed to change mine but its too small
